### PR TITLE
docs: (IAC-1307) Document update for network policy

### PR DESCRIPTION
**Pull request by riragh**
## Changes:
With the recent change introduced in PR #360, the default value for `aks_network_policy` was updated. CONFIG-VARS.md was updated to be in sync with those changes.

### CODE REVIEW

**Review by thpang — APPROVED** [review]
LGTM

**Review by jarpat — APPROVED** [review]

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -101,8 +101,8 @@ az vm image terms accept --urn Canonical:0001-com-ubuntu-pro-focal-fips:pro-fips
...
+| aks_network_policy | Sets up network policy to be used with Azure CNI. Network policy allows to control the traffic flow between pods. | string | null | Possible values are `calico` and `azure`. Network policy `azure` (Azure Network Policy Manager) is only supported for `aks_network_plugin = azure` and network policy `calico` is supported for both `aks_network_plugin` values `azure` and `kubenet`. For more details see [network policies in Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/use-network-policies).|
 | aks_network_plugin_mode | Specifies the network plugin mode used for building the Kubernetes network. Possible value is `overlay`.| string | null | When `aks_network_plugin_mode` is set to `overlay` , the `aks_network_plugin` field can only be set to `azure`. For details see Azure's documentation on: [Configure Azure CNI Overlay networking](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay).|
```
Comment (sayeun) [inline docs/CONFIG-VARS.md:106]: While we are at this, may I ask to move Possible value is `overlay` to the Notes column?  Just wanted to stay consistent.  
The other changes look good.  Thanks.

**Review by hahewlet — APPROVED** [review]
lgtm

**Review by dhoucgitter — APPROVED** [review]

File: docs/CONFIG-VARS.md (hunk 1 above)
Comment (riragh) [reply docs/CONFIG-VARS.md]: Updated as per the comment.

**Review by sayeun — APPROVED** [review]

**Pull request merged**